central/networkpolicies/graph/graph_builder_test.go

```diff
@@ -58,14 +58,29 @@ func TestMatchPolicyPeer(t *testing.T) {
 			Desc: &storage.NetworkEntityInfo_ExternalSource_{
 				ExternalSource: &storage.NetworkEntityInfo_ExternalSource{
 					Source: &storage.NetworkEntityInfo_ExternalSource_Cidr{
-						Cidr: "192.16.15.0/24",
+						Cidr: "192.168.15.0/24",
+					},
+				},
+			},
+		},
+	})
+	assert.NoError(t, err)
+	t4, err := tree.NewNetworkTreeWrapper([]*storage.NetworkEntityInfo{
+		{
+			Id:   "3",
+			Type: storage.NetworkEntityInfo_EXTERNAL_SOURCE,
+			Desc: &storage.NetworkEntityInfo_ExternalSource_{
+				ExternalSource: &storage.NetworkEntityInfo_ExternalSource{
+					Source: &storage.NetworkEntityInfo_ExternalSource_Cidr{
+						Cidr: "30.30.0.0/32",
 					},
 				},
 			},
 		},
 	})
 	assert.NoError(t, err)
-	_, _, _ = t1, t2, t3
+
+	_, _, _, _ = t1, t2, t3, t4
```

Comment:
I don't really understand what this is for. Can you try if removing it makes any difference?

Reply:
Done